src/assets/stylesheets/Button.scss

```diff
@@ -98,31 +114,31 @@
 
   // Variants
   &--primary {
-    $colours: $rpf-teal-800, $rpf-teal-600, $rpf-teal-200, $rpf-grey-600,
-      $rpf-teal-800, $rpf-teal-900;
+    $colours: var(--rpf-button-primary-background-color), var(--rpf-button-primary-background-color-active), var(--rpf-button-primary-background-color-disabled), var(--rpf-button-primary-background-color-active), var(--rpf-button-primary-background-color-focus), var(--rpf-button-primary-background-color-hover);
```

Comment:
Hi Scott. Think we might need an extra colour on this one.  We use `-rpf-button-primary-background-color-active` for both the second and fourth values, but the latter should be a grey not a teal.

I've put in a new colour  `-rpf-button-primary-color-disabled` for the fourth argument, which lines up with the `$dis-c` parameter. 

But I wanted to make a note of it here since there's a good chance I might be wrong/confused!